lib/main.dart

```diff
@@ -42,9 +42,14 @@ class _BodyWidgetState extends State<BodyWidget> {
   void openSDK() async {
     String value;
     print("opening Pngme SDK...");
+
+    // Put here your SDK token
+    // You can find it in the webconsole at admin.pngme.com under Settings > Keys
+    // If you are just checking the sample app, it is best to use the Test token
+    // Then, you can browse any sent data in the webconsole under Dashboard > Explore Data > (Select Test Environment on the right)
     try {
       value = await sdkChannel.invokeMethod("go", <String, dynamic>{
-        'sdkToken': 'XXXXXXX',
+        'sdkToken': '79037e27dd615bc4037c5b2bd3c0d359ffe94abeadb0d42aaeef9cc95eef1976f4bc7866f2414698cd831b08233b9970',
```

Comment:
Can we remove the hardcoded SDK token?

Reply:
I was waiting for somebody to complain xD I was truly thinking about keeping it, as it is the TEST token for our Pngme institution, but this could lead to misunderstandings. I will revert to the XXXXXXX